guides/source/testing.md

```diff
@@ -589,9 +589,9 @@ class BlogFlowTest < ActionDispatch::IntegrationTest
 end
 ```
 
-If you remember from earlier in the "Testing Views" section we covered `assert_select` to query the resulting HTML of a request.
+We will take a look at `assert_select` to query the resulting HTML of a request in the "Testing Views" section below. It is used for testing the response of our request by asserting the presence of key HTML elements and their content. 
```

Comment:
IMO, we should leave it. To me, at least, it fits with the flow of the section it's apart of.